.travis.yml

```diff
@@ -12,7 +12,9 @@ install:
   - npm install
 before_script: greenkeeper-lockfile-update
 script:
+  - set -e
   - npm run build
+  - (! git status --porcelain | grep -q .) || (echo "The generated values from `npm run gulp` are different from the values committed, please run `npm run build` and commit the artifact" && false) 
```

Comment:
I think that this script should be in a separate file because we'll want to use it before we release too. ()Plus it's cleaner from a travis file POV.